server/src/main/java/org/elasticsearch/index/IndexVersions.java

```diff
@@ -92,6 +92,7 @@ private static IndexVersion def(int id, Version luceneVersion) {
     public static final IndexVersion ES_VERSION_8_12 = def(8_500_004, Version.LUCENE_9_8_0);
     public static final IndexVersion NORMALIZED_VECTOR_COSINE = def(8_500_005, Version.LUCENE_9_8_0);
     public static final IndexVersion UPGRADE_LUCENE_9_9 = def(8_500_006, Version.LUCENE_9_9_0);
+    public static final IndexVersion ES_VERSION_8_13 = def(8_500_007, Version.LUCENE_9_8_0);
```

Comment:
Should this not use the new lucene version (9.9) instead?

Reply:
Oops, yes, will change.